android/app/build.gradle

```diff
@@ -232,7 +232,7 @@ android {
             keyPassword 'android'
         }
 
-        def pass = getPassword("rainbow","rainbow-android-key")
+        def pass = env.get('RAINBOW_KEY_ANDROID_PASSWORD')
```

Comment:
Can you make a PR to https://github.com/rainbow-me/rainbow-env? so everyone else can update their .env file and build release mode locally